content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/inventory_valuation_config.rst

```diff
@@ -12,11 +12,11 @@ By default, Odoo uses a periodic inventory valuation (also known as manual inven
 This method implies that the accounting team posts journal entries based on the physical inventory
 of the company, and that warehouse employees take the time to count the stock. In Odoo, this method
 is reflected inside each product category, where the :guilabel:`Costing Method` field will be set to
-`Standard Price` by default, and the :guilabel:`Inventory Valuation` field will be set to `Manual`.
+`Standard Price` by default.
 
 .. image:: inventory_valuation_config/inventory-valuation-fields.png
    :align: center
-   :alt: The Inventory Valuation fields are located on the Product Categories form.
+   :alt: The Costing Method field is located on the Product Categories form.
 
 Alternatively, automated inventory valuation is an integrated valuation method that updates the
```

Comment:
I'm thinking a note could be added (could be a copy/paste of line #78) here about the need to have the Accounting app installed. As of right now, and considering the user follows the flow in the PR (that is, only having inventory installed - without accounting), they're introduced to the concept of "Automatic Inventory Valuation" without seeing the option in their DB. A note would make it clearer for both:

1) Users without Accounting installed at that point in time, as they would know about this other feature explained to them, but also why it's not available to them in their DB
2) Users with Accounting installed: so far, this seems to pertain only to Inventory users (everything revolves around Inventory) when it is also meant for Accounting users. Maybe a :ref: could be added to line #78 for those looking for automated valuation.